### PR TITLE
Feature/mds 1577

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**
Completed the warning for users on mobile devices.

### CODE REVIEW

File: frontend/src/components/Home.js [hunk 1]
```diff
@@ -69,6 +74,12 @@ export class Home extends Component {
         </div>
         {this.state.isIE && <WarningBanner onClose={this.handleIEClose} type="IE" />}
         {this.state.isTest && <WarningBanner type="test" />}
+        <MediaQuery maxWidth={769}>
```
Comment (mmumby) [inline frontend/src/components/Home.js]: update to be 500, or 501. 
769 is tablet width

File: frontend-public/src/App.js [hunk 2]
```diff
@@ -12,12 +13,16 @@ import WarningBanner from "@/components/common/WarningBanner";
...
 
 class App extends Component {
-  state = { isIE: true };
+  state = { isIE: true, isMobile: true };
 
   componentDidMount() {
     this.setState({ isIE: detectIE() });
   }
 
+  handleMobileWaringClose = () => {
```
Comment (NWCalvank) [inline frontend-public/src/App.js]: Misspelling: `handleMobileWarningClose`

File: frontend/src/components/Home.js (hunk 1 above)
Comment (thegentlemanphysicist) [reply frontend/src/components/Home.js]: done

File: frontend-public/src/App.js (hunk 2 above)
Comment (thegentlemanphysicist) [reply frontend-public/src/App.js]: fixed

**Pull request merged**